### ISSUE TITLE
multiple ##c.var

### ISSUE THREAD

**Issue by matthieugomez** [sergiocorreia/reghdfe#43]
```
sysuse nlsw88.dta
reghdfe wage i.industry##c.age i.union##c.age, a(race)
error: there are repeated variables: <age>
r(198);
areg wage i.industry##c.age i.union##c.age, a(race)
```

**Comment by sergiocorreia** [comment]
Nice catch!
I'll tackle it later on the week

**Comment by sergiocorreia** [comment]
Just as an update; this bug is closely related to #35 and #22 (due to the way the `syntax` command expands the variable).

I'll add an update to fix these three issues, and *maybe* also #28 which is also related to factors

**Comment by sergiocorreia** [comment]
Works on dev version; closed (after more than a year!)
